.ci/install_kata_image.sh

```diff
@@ -80,7 +80,11 @@ build_image_for_cc () {
 		[ "${osbuilder_distro:-ubuntu}" == "ubuntu" ] || \
 			die "The only supported image for Confidential Containers is Ubuntu"
 
-		build_static_artifact_and_install "rootfs-image"
+		if [ "${TEE_TYPE}" == "tdx" ]; then
```

Comment:
What about replacing the whole `if` thing with

```
build_static_artifact_and_install ${TEE_TYPE/%/-}rootfs-image
```
or something like that. The benefit is that when we have `TEE_TYPE` set to something else, we get that for free.